firmware/user/display/oled.c

```diff
@@ -202,6 +202,213 @@ void drawPixel(int16_t x, int16_t y, color c)
     }
 }
 
+
+/**
+ * Set/clear/invert a single pixel.
+ *
+ * This intentionally does not have ICACHE_FLASH_ATTR because it may be called often
+ *
+ * @param x Column of display, 0 is at the left
+ * @param y Row of the display, 0 is at the top
+ */
+
+void drawPixelFastWhite( int x, int y )
```

Comment:
I think this and `speedyWhiteLine()` need to get copied to the emulator's display code.

Reply:
Changing Name